xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/serverinfo/ServerInfo.java

```diff
@@ -0,0 +1,205 @@
+   *   otherwise.
+   */
+  @Value.Derived
+  @JsonIgnore
+  default boolean isLedgerInCompleteLedgers(final UnsignedLong ledgerIndex) {
+    return this.completeLedgers().stream().anyMatch(range -> range.contains(ledgerIndex));
+  }
+
+  /**
+   * The number of times (since starting up) that this server has had over 250 transactions waiting to be processed at
+   * once. A large number here may mean that your server is unable to handle the transaction load of the XRP Ledger
+   * network. For detailed recommendations of future-proof server specifications, see
+   * <a href="https://xrpl.org/capacity-planning.html">Capacity Planning</a>.
+   *
+   * @return An optionally-present {@link String}.
+   */
+  @JsonProperty("jq_trans_overflow")
+  Optional<String> jqTransOverflow();
+
+  /**
+   * The load-scaled open ledger transaction cost the server is currently enforcing, as a multiplier on the base
+   * transaction cost. For example, at 1000 load factor and a reference transaction cost of 10 drops of XRP, the
+   * load-scaled transaction cost is 10,000 drops (0.01 XRP). The load factor is determined by the highest of the
+   * individual server's load factor, the cluster's load factor, the open ledger cost and the overall network's load
+   * factor. Per xrpl.org docs, this field is optionally present in any server response and may be omitted because it is
+   * only returned when requested via an admin host/port.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the load factor.
+   */
+  @JsonProperty("load_factor")
+  Optional<BigDecimal> loadFactor();
+
+  /**
+   * (Admin only) Public key used by this node to sign ledger validations. This validation key pair is derived from the
+   * {@code [validator_token]} or {@code [validation_seed]} config field. Per xrpl.org docs, this field is optionally
+   * present in any server response and may be omitted because it is only returned when requested via an admin
+   * host/port.
+   *
+   * @return A {@link String} containing the validator's public key.
+   */
+  @JsonProperty("pubkey_validator")
+  Optional<String> publicKeyValidator();
+
+  /**
+   * Information about the most recent fully-validated ledger. If the most recent validated ledger is not available, the
+   * response omits this field and includes {@link #closedLedger()} instead. Per xrpl.org docs, this field is optionally
+   * present in any server response and may be omitted.
+   *
+   * @return An optionally-present {@link ServerInfoLedger} representing the latest validated ledger.
+   */
+  @JsonProperty("validated_ledger")
+  Optional<ServerInfoLedger> validatedLedger();
+
+  /**
+   * Minimum number of trusted validations required to validate a ledger version. Some circumstances may cause the
+   * server to require more validations.
+   *
+   * @return An {@link UnsignedInteger} representing the quorum.
+   */
+  @JsonProperty("validation_quorum")
```

Comment:
this is optional in clio. clio tries to get the info from rippled, but if rippled does not respond for some reason, the info is not included